dkpro-core-lancaster-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/lancaster/LancasterStemmer.java

```diff
@@ -0,0 +1,144 @@
+package de.tudarmstadt.ukp.dkpro.core.lancaster;
+
+import de.tudarmstadt.ukp.dkpro.core.api.featurepath.FeaturePathAnnotatorBase;
+import de.tudarmstadt.ukp.dkpro.core.api.featurepath.FeaturePathException;
+import de.tudarmstadt.ukp.dkpro.core.api.segmentation.type.Stem;
+import de.tudarmstadt.ukp.dkpro.core.api.segmentation.type.Token;
+import org.apache.commons.lang.StringUtils;
+import org.apache.uima.analysis_engine.AnalysisEngineProcessException;
+import org.apache.uima.cas.CAS;
+import org.apache.uima.cas.FSIterator;
+import org.apache.uima.cas.Feature;
+import org.apache.uima.cas.Type;
+import org.apache.uima.cas.text.AnnotationFS;
+import org.apache.uima.cas.text.AnnotationIndex;
+import org.apache.uima.fit.descriptor.ConfigurationParameter;
+import org.apache.uima.jcas.JCas;
+
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Set;
+
+/**
+ * This Paice/Husk Lancaster stemmer implementation only works with the English language so far.
+ *
+ * TODO: The wrapped Lancaster stemmer of the Smile API should be enhanced to support custom rules files.
```

Comment:
Todo should be outside the JavaDoc comment as the JavaDoc comment is used to generate the reference documentation.